first session/assignment/shoppingmall.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>나만의 쇼핑몰</title>
+    <link rel="stylesheet" href="shoppingmall.css">
+</head>
+<body>
+    <header>   
+        <div class="header-icon">🛍️</div>
+        <h1>민준이의 쇼핑몰</h1>
+        <!-- <div class="menu-bar">☰</div> -->
+        <!--<img src="./static/img/menu-bar.png" alt="menu bar">-->
+        <div class="hamburger-menu">
+            <button class="hamburger-icon" onclick="toggleMenu()">
+                ☰
+            </button>
+                   
+        </div>
+    </header>
+        <div class>
+        
+    </div>
+    <nav>
+        <ul>
+            <li><a href="#all">전체</a></li>
+            <li><a href="#category1">운동용품</a></li>
+            <li><a href="#category2">여가생활</a></li>
+            <li><a href="#category3">의류</a></li>
+        </ul>
+    </nav>
+    <main>
+        <div>
+            <h2>전체상품</h2>
+            <section id="all">
```

Comment:
위에 언급했듯이 하나의 태그에는 id와 class를 동시에 지정할 수 있습니다!!
```suggestion
            <section id="all" class="category">
```